pkg/antctl/command_definition.go

```diff
@@ -90,6 +91,11 @@ var groupCommands = map[commandGroup]*cobra.Command{
 		Short: "Sub-commands of multi-cluster feature",
 		Long:  "Sub-commands of multi-cluster feature",
 	},
+	upgrade: {
+		Use:   "upgrade",
+		Short: "Upgrade Antrea CRD resources to the latest API version",
+		Long:  "Upgrade Antrea CRD resources to the latest API version",
```

Comment:
This is the description of the "upgrade" command, not "upgrade api-version", it should be more generic, like "Sub-commands for upgrade operations"

Reply:
Done